src/Permissions.ts

```diff
@@ -35,10 +36,19 @@ export class Permissions {
    * @param action - `Actions` enum value
    * @param forSpace - Space object that must contain `{ teams: Team[] }`
    * @param forUserId - string userId
+   * @param forRevokeInvitation - Additional information to determine if a user can revoke invitation
    * @returns boolean
    * @throws "Could not get role for space with no teams" exception
    */
-  canSpace(action: Actions, forSpace: Space, forUserId: string) {
+  canSpace(
+    action: Actions,
+    forSpace: Space,
+    forUserId: string,
+    forRevokeInvitation?: {
+      invitationId: string;
+      invitationsCreatedByUserId: Invitation[];
```

Comment:
Why don't we just require "createdById" instead of all invitations?

Reply:
that's true...

Reply:
Updated https://github.com/lensapp/lens-platform-sdk/pull/85/commits/b2dcc07c974d7049508a46a1564bfc7c8a16fcb1